unity-client/Assets/Scripts/MainScripts/DCL/Bridges/BuilderProjectsPanelBridge/Types.cs

```diff
@@ -0,0 +1,8 @@
+public class SceneDataUpdatePayload
+{
+    public string name;
+    public string description;
+    public string[] requiredPermissions;
+    public bool isMatureContent;
+    public bool allowVoiceChat;
+}
```

Comment:
Should the scene data also contain the dimension (1x1, 2x2)?

Reply:
I will probably add that in other PR when need it... updating scenes dimensions it's something that I didn't have to do, yet